fwoa-tools/src/migrationImport.ts

```diff
@@ -320,9 +319,22 @@ async function deleteResourcesInBundle(deletePaths: string[]): Promise<void> {
 }
 
 (async () => {
```

Comment:
For the code inside `async` you can ignore them from coverage by putting `/* istanbul ignore next */` on line 320. Then you can move most of the logic in `async` to it's own method like done [here](https://github.com/aws-solutions/fhir-works-on-aws/blob/89d0fca849259feda69c76232296814d46d0c066/fwoa-tools/src/migrationExport.ts#L146)

Reply:
Updated